src/components/Main/Button/index.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components"
+const Button = ({action, children}) => {
```

Comment:
`action` is not used anywhere in this component

```suggestion
const Button = ({ children }) => {
```